src/main/java/core/basesyntax/db/InvalidDataUser.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.db;
+
+public class InvalidDataUser extends RuntimeException{
```

Comment:
Виняток також має вказувати на те що це він є тому потрібно добавляти у назву Exception
```suggestion
public class InvalidUserException extends RuntimeException{
```